R/downloadFile.R

```diff
@@ -239,17 +242,17 @@ downloadFile <- function(id,
                 filename = shiny::reactive({paste(rootname(), "txt", sep = ".")}),
                 content = function(file) {
                     if (!is.null(datafxns)) {
-                        writeFile("txt", datafxns$txt(), file, logger,
+                        writeFile("txt", datafxns$txt(), file, row_names, logger,
                                   shiny::reactive({paste(rootname(), "txt", sep = ".")}))
                     }
                 })
 
             # filename is expected to be a reactive expression
-            writeFile <- function(type, data, file, logger, filename) {
+            writeFile <- function(type, data, file, show_rownames, logger, filename) {
```

Comment:
can we take the chance and update this function documentation and document what these paramters are used for?